frontend/src/views/Blacksmith.vue

```diff
@@ -996,7 +998,9 @@ export default Vue.extend({
     },
 
     setChosenElement(ele: any, i: number) {
-      this.selectedElement = i;
+      if(i === this.selectedElement) this.selectedElement = null;
+      else this.selectedElement = i;
```

Comment:
none descriptive variable names is my personal pet peeve! Can you rename it what it is storing for easy reading <3